SonarQube.Bootstrapper/Resources.resx

```diff
@@ -135,6 +135,9 @@
   <data name="ERROR_FailedToUpdateRunnerBinaries" xml:space="preserve">
     <value>Failed to update the SonarQube MSBuild Runner binaries. Check the server url, verify that the C# plugin is correctly installed on the SonarQube server and that the SonarQube server has been restarted.</value>
   </data>
+  <data name="ERROR_PostProcessExeNotFound" xml:space="preserve">
+    <value>Post-processor executable not found '{0}'. Please check that pre-processing was executed in the current directory.</value>
```

Comment:
Might be worth mentioning the build step too: if the user has executed the post-processing from the wrong directory then they might have built the wrong directory too e.g.
"Please check that the pre-processing step, the build step and the post-processing step were all executed in the same directory."